src/custom-hook/useToastify.jsx

```diff
@@ -0,0 +1,23 @@
+import { toast } from "react-toastify";
+
+const useToastify = () => {
+	const showToast = (toastText="Operation Successfull", toastTheme="success") => {
+		const notify = () => {
+			toast[toastTheme](toastText, {
+                theme: "colored",
```

Comment:
Just so you don't miss it

Reply:
This is very weird. I ran prettier and looks fine in the VS code.
![image](https://user-images.githubusercontent.com/51514137/161318059-dcf0668d-3788-492a-bb88-2c8a010d3ba3.png)